### PR TITLE
adam's first help

### PR DESCRIPTION
**Pull request by tautme**
I'm helping

### CODE REVIEW

**Review by danielngilmour — CHANGES_REQUESTED** [review]
Thanks, but no thanks.

File: index.html [hunk 1]
```diff
@@ -8,4 +9,4 @@
 <body>
     <p>Hello World. Where is the rice?</p>
```
Comment (danielngilmour) [inline index.html:9]: All of this is incorrect.

**Comment by tautme** [comment]
I am working as hard as I can....

File: index.html (hunk 1 above)
Comment (tautme) [reply index.html:9]: I will examine and decide to follow instruction.

**Pull request merged**